app/__init__.py

```diff
@@ -245,7 +245,7 @@ def update_sent_state(sender=None, headers=None, **kwargs):
 
 @app.errorhandler(500)
 def internal_server_error(error):
-    if current_app.config['PROPOGATE_ERROR'] is True:
+    if current_app.config['DEVELOPMENT'] is True:
```

Comment:
This is redundant, propagate error is already true for development config.

https://github.com/fossasia/open-event-server/blob/8f2de81dbbfe179770f160f59fc365316ffed660/config.py#L119